### PR TITLE
Improve SEO

### LINKED ISSUE #209: SEO: Document does not have a meta description
On minetest.net, a Lighthouse audit produces this error:

> Document does not have a meta description
> Meta descriptions may be included in search results to concisely summarize page content.

See also: https://web.dev/meta-description/

### PR DESCRIPTION
**Pull request by Panquesito7**
Things added/changed:

- Improve SEO.
  - Closes #209.

### CODE REVIEW

File: _includes/head.html [hunk 1]
```diff
@@ -26,6 +26,12 @@
 <meta name="msapplication-TileColor" content="#ffffff">
 <meta name="msapplication-TileImage" content="{{ '/media/favicon/144x144.png' | relative_url }}">
 <meta name="theme-color" content="#ffffff">
+<meta name="description" content="An open source voxel game engine. Play one of our many games, mod a game to your liking, make your own game, or play on a multiplayer server.">
+<meta property="og:url" content="https://www.minetest.net" />
+<meta property="og:type" content="website" />
+<meta property="og:locale" content="en_US" />
+<meta property="og:image" content="https://raw.githubusercontent.com/minetest/minetest.github.io/master/favicon.ico" />
```
Comment (sfan5) [inline _includes/head.html:33]: should link to `/favicon.ico`

File: _includes/head.html [hunk 2]
```diff
@@ -26,6 +26,12 @@
...
+<meta property="og:description" content="An open source voxel game engine. Play one of our many games, mod a game to your liking, make your own game, or play on a multiplayer server." />
```
Comment (sfan5) [inline _includes/head.html:34]: are we sure these headers should be added to *every* page in this form?

Comment (rubenwardy) [reply _includes/head.html:34]: it should not, this should only be added to the homepage

Comment (sfan5) [reply _includes/head.html:34]: Icon and such could stay for every page as long as it uses the correct title.

Comment (rubenwardy) [reply _includes/head.html:34]: The image should be a .png, not a .ico as well. It's not a fav icon.

Also, the locale should be specified in <html>, the og:locale is pointless

Comment (Panquesito7) [reply _includes/head.html:34]: I will move these only to the `index.html`. Sounds good?

**Pull request closed without merging**